infogami/infobase/tests/test_save.py

```diff
@@ -72,20 +76,21 @@ def test_new(self):
 
     def test_existing(self):
         def insert(doc, revision, created, last_modified):
-            id =  db.insert('thing', key=doc['key'], latest_revision=revision, created=created, last_modified=last_modified)
+            id = db.insert('thing', key=doc['key'], latest_revision=revision, created=created, last_modified=last_modified)
             db.insert('data', seqname=False, thing_id=id, revision=revision, data=simplejson.dumps(doc))
 
-        created = datetime.datetime(2010, 01, 01, 01, 01, 01)            
+        created = datetime.datetime(2010, 01, 01, 01, 01, 01)
```

Comment:
The GitHub Editor is trying to help you here.  01 is a syntax error in Python 3.  We can these later if you want.

Reply:
I was going to leave it for later, but since I had to rebase to resolve the conflicts, I fixed them.